### PR TITLE
Fix/optimize rendering

### PR DESCRIPTION
**Pull request by felquis**
This pull request:

* Bumps ejs from 1.0.0 to 2.5.5
* Begins caching template file and rendering function

Before this upgrade generating 50k boletos took 100s. Afterwards, it took 56s.

### CODE REVIEW

File: lib/boleto.js [hunk 1]
```diff
@@ -84,9 +79,10 @@ Boleto.prototype.renderHTML = function(callback) {
 
 	renderOptions['boleto']['linha_digitavel_hash'] = hashString(renderOptions['boleto']['linha_digitavel']).toString();
 
-	var html = ejs.render(content.toString(), renderOptions);
-
-	callback(html);
+	var html = ejs.renderFile(__dirname + '/../assets/layout.ejs', renderOptions, {
+    cache: true
+  }, (err, html) => {
```
Comment (joaolucasl) [inline lib/boleto.js]: Arrow functions may break compat with older Node versions :)

Comment (theiostream) [inline lib/boleto.js]: :+1:

**Comment by theiostream** [comment]
Foi rebaseado com a master e passou nos testes. @felquis quando puder, sinta-se livre pra mergear e publicar.

**Comment by felquis** [comment]
Não passou nos testes...

**Comment by felquis** [comment]
Fixing these little poops 

captura de tela de 2017-02-03 13-26-14

**Pull request merged**